feat(MultiTwitch): add presence

sweezyio opened the pull request:
## Description 
Presence for [MultiTwitch](https://multitwitch.tv), a platform allowing users to watch any number of Twitch streams simultaneously all on the same page. 

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>
- ![Side-by-Side comparison](https://user-images.githubusercontent.com/33377824/221717566-a12c06ee-a47d-4322-9eab-8d998dc95ad9.PNG)
- ![Profile Image 1](https://user-images.githubusercontent.com/33377824/221717412-a938be68-e893-488a-978b-96c2cf13a581.png)
- ![Profile Image 2](https://user-images.githubusercontent.com/33377824/221717523-4cc3f002-ccd3-47ad-9eb4-e490044bb108.png)
</details>

RisingSunLight42 commented:
Revert your merge and fix your pull request description

RisingSunLight42 reviewed (commented):
Really low quality I think. You could add a slideshow to display every streams on Discord to make the presence more interesting.

Bas950 commented on websites/M/MultiTwitch/metadata.json:
```diff
@@ -0,0 +1,27 @@
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "181381194070491136",
+		"name": "3Tau#7777"
```
```suggestion
		"name": "3Tau"
```

Bas950 commented on websites/M/MultiTwitch/metadata.json:
```diff
@@ -0,0 +1,27 @@
+		"multitwitch.tv",
+		"www.multitwitch.tv"
+	],
+	"regExp": "(www.)?(multitwitch.tv)",
```
```suggestion
```
Seems like a useless regex, the url should be enough

Bas950 reviewed (changes requested):
Could you expand it to show more data/pages? Cause it would be denied for low quality.

sweezyio commented:
I updated the presence to use a slideshow to display all of the streams that are currently being watched (and an option to disable that in favor of a simple timestamp). And I added the option to include a button in the presence to open the current URL from Discord.

Updated presence with settings:
![premid settings](https://user-images.githubusercontent.com/33377824/224173789-1381348b-b549-4e3e-80e3-6ed180af2bae.PNG)

sweezyio reviewed (commented):
Updated to show more detail
